.gitreview

```diff
@@ -1,4 +1,4 @@
 [gerrit]
 host=gerrit.sqlalchemy.org
 project=sqlalchemy/sqlalchemy
-defaultbranch=master
+defaultbranch=rel_2_0
```

Comment:
is it correct?

Reply:
if the PR is against master then that should stay at "master".